chat-client/src/client/mynahUi.ts

```diff
@@ -46,11 +46,6 @@ export const handleChatPrompt = (
             mynahUi.updateStore(tabId, {
                 chatItems: [],
             })
-
-            mynahUi.updateStore(tabId, {
-                loadingChat: false,
-                promptInputDisabledState: false,
```

Comment:
From testing in VS, I found that I need these two at least for. But it can be combined with the above `updateStore`

Reply:
I'll test it out a little more